packages/tx-construction/test/tx-builder/TxBuilder.test.ts

```diff
@@ -302,6 +304,11 @@ describe('GenericTxBuilder', () => {
       expect(outputBuilder.toTxOut().datumHash).toEqual(datumHash);
     });
 
+    it('can set handle', () => {
```

Comment:
I think we're missing a piece of essential functionality
```suggestion
    describe('build()', () => {
      it('resolves handle to an address');
      it('failing to resolve handle rejects with UnableToResolveHandleError');
    });
    it('can set handle', () => {
```

Reply:
There is no functionality for that yet. It will come in the next ticket so it makes more sense to add the tests when adding the actual TxBuilder handle resolution code.

Reply:
caf857dd9cbf3d89392f48f77bb91a1c55696edb should solve this